### ISSUE TITLE
Flutter 2.5.0 breaks fast_flutter_driver_tool

### ISSUE THREAD

**Issue by DaveBubenik** [tomaszpolanski/fast_flutter_driver#119]
When attempting to use the fastdriver tool on windows, I see the following error:
```
Unhandled exception:
Bad state: Future already completed
```
This is occurring because the latest version of flutter prints "is available at" multiple times when starting the app in debug mode. The tool can be fixed by modifying the nativeMatch variable in the _buildAndRun method in testing.dart:
```
final nativeMatch = RegExp(
              'An Observatory debugger and profiler on .* is available at: (http://.*/)')
          .firstMatch(line);
```

**Comment by tomaszpolanski** [comment]
Hey @DaveBubenik ,
thanks for reporting that, will look into it

**Comment by tomaszpolanski** [comment]
Resolved in `fast_flutter_driver_tool` 3.0.0
